### PR TITLE
chore: add discussion templates

### PR DESCRIPTION
**Pull request by knqyf263**
## Description
Sort our issues/discussions

You can see what it looks like [here](https://github.com/knqyf263/trivy/issues/new/choose).

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [x] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: .github/DISCUSSION_TEMPLATE/q-a.yml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        - Virtual Machine Image
+        - Kubernetes
+        - AWS
+    validations:
+      required: false
+  - type: dropdown
+    attributes:
+      label: Scanner
+      description: Which scanner are you using?
+      options:
```
Comment (chen-keinan) [inline .github/DISCUSSION_TEMPLATE/q-a.yml:34]: maybe there q-a on `sbom`

File: .github/DISCUSSION_TEMPLATE/q-a.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    attributes:
+      label: Question
+      description: What kind of problem are you facing? Or, what questions do you have?
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Target
+      description: Which target are you scanning?
+      options:
```
Comment (chen-keinan) [inline .github/DISCUSSION_TEMPLATE/q-a.yml:21]: no `server`?

File: .github/DISCUSSION_TEMPLATE/ideas.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    attributes:
+      label: Description
+      description: Describe your idea.
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Target
+      description: Which target is your idea related to?
+      options:
```
Comment (chen-keinan) [inline .github/DISCUSSION_TEMPLATE/ideas.yml:21]: no `server` ?

File: .github/DISCUSSION_TEMPLATE/false-detection.yml [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+      description: Output of run with `--version`
+      placeholder: "$ trivy --version"
+      render: bash
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Checklist
+      options:
+        - label: Read [the documentation regarding wrong detection](https://aquasecurity.github.io/trivy/latest/community/contribute/discussion/#false-detection)
```
Comment (chen-keinan) [inline .github/DISCUSSION_TEMPLATE/false-detection.yml:86]: maybe link is broken?

Comment (knqyf263) [reply .github/DISCUSSION_TEMPLATE/false-detection.yml:86]: Yes, the page will be generated after this PR gets merged. I forgot to add a note.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 5]
```diff
@@ -0,0 +1,14 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Report a false detection
+    url: https://github.com/knqyf263/trivy/discussions/new?category=false-detection
```
Comment (knqyf263) [inline .github/ISSUE_TEMPLATE/config.yml:4]: TODO: replace the link

**Comment by chen-keinan** [comment]
lgtm , I have added couple of questions

File: .github/DISCUSSION_TEMPLATE/q-a.yml (hunk 1 above)
Comment (knqyf263) [reply .github/DISCUSSION_TEMPLATE/q-a.yml:35]: Nice catch! I've added SBOM.

File: .github/DISCUSSION_TEMPLATE/ideas.yml (hunk 3 above)
Comment (knqyf263) [reply .github/DISCUSSION_TEMPLATE/ideas.yml:21]: Added "Mode".

File: .github/DISCUSSION_TEMPLATE/q-a.yml (hunk 2 above)
Comment (knqyf263) [reply .github/DISCUSSION_TEMPLATE/q-a.yml:21]: Added!

**Review by DmitriyLewen — APPROVED** [review]
LGTM

**Pull request merged**